psychopy/iohub/client/wintab.py

```diff
@@ -140,8 +140,8 @@ def velocity(self):
         return self._velocity
 
     @property
-    def accelleration(self):
```

Comment:
This one is a bit scary - spellchecking the docs is all good, but this has the potential to actually break something. It's better to create an alias rather than changing the existing method, so users can use either.

Reply:
I agree. See my initial comment:
> Please also note that the following changes break the API by modifying property names. Not sure how stable that part of the API is supposed to be. If that's a problem, please do tell me, I can revert these specific changes and later propose a more complex change that would preserve backwards compatibility:

Reply:
Additionally, as a proof that the change is intrusive and error-prone, I had forgot to change `self._accelleration`!

Such a change should go in its own PR and make sure the code is backwards compatible. I'll do that after this PR.